examples/stable_diffusion.py

```diff
@@ -606,7 +606,7 @@ def __init__(self):
   parser = argparse.ArgumentParser(description='Run Stable Diffusion', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
   parser.add_argument('--steps', type=int, default=5, help="Number of steps in diffusion")
   parser.add_argument('--prompt', type=str, default="a horse sized cat eating a bagel", help="Phrase to render")
-  parser.add_argument('--out', type=str, default="/tmp/rendered.png", help="Output filename")
+  parser.add_argument('--out', type=str, default=os.path.join(os.path.dirname(os.path.abspath(__file__)), "rendered.png"), help="Output filename")
```

Comment:
Why? Why not just cwd?

Reply:
Just have the whole script change to the `examples/` folder and keep this as `rendered.png`?